kong/api/routes/cluster.lua

```diff
@@ -33,7 +33,10 @@ return {
         return responses.send_HTTP_BAD_REQUEST("Missing node \"name\"")
       end
 
-      local _, err = singletons.serf:invoke_signal("force-leave", {self.params.name})
+      local inspect = require "inspect"
+      print(inspect(self.params.name))
```

Comment:
print? where does this go and will it include any secure/confidential information?